modules/doc/content/newsletter/2024/2024_11.md

```diff
@@ -1,16 +1,112 @@
 # MOOSE Newsletter (November 2024)
 
-!alert! construction title=In Progress
-This MOOSE Newsletter edition is in progress. Please check back in December 2024
-for a complete description of all MOOSE changes.
-!alert-end!
-
 ## MOOSE Improvements
 
-## libMesh-level Changes
+### Default support for multiple solver systems in the Steady and Transient executioners
+
+The [Steady.md] and [Transient.md] executioner, as well as any `Executioner` relying on a [FEProblemSolve.md] solve object
+can now solve multiple solver systems, e.g. nonlinear or linear systems. A variable can be placed in a solver system
+using the `solver_sys` parameter of the `Variable`.
+
+The solve of multiple systems can be iterated using the [!param](/Executioner/Transient/multi_system_fixed_point) parameter.
+A [Convergence object](syntax/Convergence/index.md) must then be specified to the `Executioner` to assess the convergence
+of the multi-system fixed point iteration.
+
+This support was also extended to the [Physics system](syntax/Physics/index.md), in which the `Physics` equations can be
+set to be solved in their own solver system
+
+### Interface for finding data files
+
+Parameters with the string-derived type `DataFileName` will now search the application's data folder, as well as the
+local simulation folder, for the files mentioned in the parameter. They will then return the absolute path to those files.
+This is intended to simplify the inclusion of data in an application, data which can be referred to in the input files.
+
+### New ChainControl system for handling complex/chained Control logic
+
+The new [ChainControl system](syntax/ChainControls/index.md) was added to MOOSE. It uses `ControlData` which can be of arbitrary
+types, and which can be operated on by each `ChainControl` object. This enables creating limiters, PID controllers, etc,
+and chaining these operations, to control parameters in the input file.
+This was inspired by the `ControlLogic` system in the thermal-hydraulics module.
```

Comment:
```suggestion
The new [ChainControl system](syntax/ChainControls/index.md) was added to MOOSE. It uses `ChainControlData` which can be of arbitrary
types, and which can be operated on by each `ChainControl` object. This enables creating limiters, PID controllers, etc.,
and chaining these operations, to control parameters in the input file.
This will replace the `ControlLogic` system in the thermal-hydraulics module after a transition period.
```